ui/tickets_page.go

```diff
@@ -0,0 +1,477 @@
+}
+
+func (pg *ticketPage) LayoutTicketList(gtx layout.Context, common pageCommon) layout.Dimensions {
+	return layout.UniformInset(values.MarginPadding0).Layout(gtx, func(gtx C) D {
+		return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+			layout.Flexed(1, func(gtx C) D {
+				margin := values.MarginPadding20
+				return layout.Inset{Left: margin, Right: margin, Bottom: margin}.Layout(gtx, func(gtx C) D {
+					return layout.Center.Layout(gtx, func(gtx layout.Context) layout.Dimensions {
+						if *pg.tickets == nil {
+							return layout.Dimensions{}
+						}
+						walletID := common.info.Wallets[pg.walletsDropdown.SelectedIndex()].ID
+						tickets := (*pg.tickets).Confirmed[walletID]
+						if len(tickets) == 0 {
+							return common.theme.Body2("No ticket").Layout(gtx)
+						}
+
+						return pg.ticketList.Layout(gtx, len(tickets), func(gtx C, index int) D {
+							return pg.ticketRowInfo(gtx, common, tickets[index])
+						})
+					})
+				})
+			}),
+		)
+	})
+}
+
+func (pg *ticketPage) LayoutUnconfirmedPurchased(gtx layout.Context, common pageCommon) layout.Dimensions {
+	return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+		layout.Rigid(func(gtx C) D {
+			tickets := (*pg.tickets).Unconfirmed[common.info.Wallets[pg.walletsDropdown.SelectedIndex()].ID]
+			margin := values.MarginPadding20
+			return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+				layout.Flexed(1, func(gtx layout.Context) layout.Dimensions {
+					return layout.Inset{Left: margin, Right: margin, Bottom: margin}.Layout(gtx, func(gtx C) D {
+						return layout.Center.Layout(gtx, func(gtx layout.Context) layout.Dimensions {
+							return pg.unconfirmedList.Layout(gtx, len(tickets), func(gtx C, index int) D {
+								return pg.ticketRowInfo(gtx, common, tickets[index])
+							})
+						})
+					})
+				}),
+			)
+		}),
+	)
+}
+
+func (pg *ticketPage) ticketRowHeader(gtx layout.Context, common pageCommon) layout.Dimensions {
+	txt := common.theme.Label(values.MarginPadding15, "#")
+	txt.MaxLines = 1
+	txt.Color = common.theme.Color.Hint
+	return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+		layout.Flexed(1, func(gtx layout.Context) layout.Dimensions {
+			margin := values.MarginPadding20
+			return layout.Inset{Right: margin, Left: margin}.Layout(gtx, func(gtx layout.Context) layout.Dimensions {
+				return layout.Center.Layout(gtx, func(gtx layout.Context) layout.Dimensions {
+					return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+						layout.Rigid(func(gtx C) D {
+							gtx.Constraints.Min.X = gtx.Px(values.MarginPadding60)
+							return txt.Layout(gtx)
+						}),
+						layout.Rigid(func(gtx C) D {
+							gtx.Constraints.Min.X = gtx.Px(values.MarginPadding120)
+							txt.Text = "Date (UTC)"
+							return txt.Layout(gtx)
+						}),
+						layout.Rigid(func(gtx C) D {
+							gtx.Constraints.Min.X = gtx.Px(values.MarginPadding120)
+							txt.Text = "Status"
+							return txt.Layout(gtx)
+						}),
```

Comment:
`values.MarginPadding120` repeated twice. Kindly declare this a variable and reuse. `m120 := values.MarginPadding120`
```suggestion
		layout.Rigid(func(gtx C) D {
			gtx.Constraints.Min.X = gtx.Px(m120)
			txt.Text = "Date (UTC)"
			return txt.Layout(gtx)
		}),
		layout.Rigid(func(gtx C) D {
			gtx.Constraints.Min.X = gtx.Px(m120)
			txt.Text = "Status"
			return txt.Layout(gtx)
		}),
```